storage_controller/src/service.rs

```diff
@@ -2816,8 +2819,11 @@ impl Service {
 
             // Pick an arbitrary node to use for remote deletions (does not have to be where the tenant
             // was attached, just has to be able to see the S3 content)
-            let node_id =
-                scheduler.schedule_shard::<AttachedShardTag>(&[], &ScheduleContext::default())?;
+            let node_id = scheduler.schedule_shard::<AttachedShardTag>(
```

Comment:
nit: as the schedule_shard prototype grows in complexity, maybe now is a good time to give scheduler some "any_available_node()" helper that wraps the invocation we use when we just want any node to do some S3 work on

Reply:
https://github.com/neondatabase/neon/pull/9083/commits/ab411e3bf1de6a41b2a6d92dcba60c6be89705d9